<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="generator" content="pandoc" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <meta name="author" content="Shazia Ruybal-Pesántez" />
    <meta name="date" content="2023-06-01" />
    <title>Shazia’s resume</title>

    
        <link href="shazia_resume_files/font-awesome-6.4.0/css/all.min.css" rel="stylesheet" />
        <link href="shazia_resume_files/font-awesome-6.4.0/css/v4-shims.min.css" rel="stylesheet" />
        <link href="shazia_resume_files/paged-0.20/css/resume.css" rel="stylesheet" />
        <script src="shazia_resume_files/paged-0.20/js/config.js"></script>
        <script src="shazia_resume_files/paged-0.20/js/paged.js"></script>
        <script src="shazia_resume_files/paged-0.20/js/hooks.js"></script>
    
        <link rel="stylesheet" href="css/edits.css" type="text/css" />
    
  </head>
  <body>

<div id="aside" class="section level1">
<h1>Aside</h1>
<p><img src="shazia.JPG" style="width:60.0%" alt="Shazia" /></p>
<div id="contact" class="section level2">
<h2>Contact Info</h2>
<ul>
<li><i class="fa fa-envelope"></i> <a href="mailto:s.ruybal@imperial.ac.uk" class="email">s.ruybal@imperial.ac.uk</a></li>
<li><i class="fa fa-github"></i> <a href="https://github.com/shaziaruybal">github.com/shaziaruybal</a></li>
<li><i class="fa fa-globe"></i> <a href="https://shaziaruybal.com">shaziaruybal.com</a></li>
<li><i class="fa fa-twitter"></i> <a href="https://twitter.com/drshaziaruybal"><span class="citation">@DrShaziaRuybal</span></a></li>
</ul>
<p><br>
I am an experienced <code>R</code> user for data processing, vizualization, communication, reports/dashboards, shiny apps, workshop development/organization and am passionate about open science and the development of digital tools for research. See below for more details.</p>
</div>
<div id="skills" class="section level2">
<h2>Skills</h2>
<ul>
<li><p><i class="fa fa-chart-line"></i> Full experience in genetic/genomic sequencing data analysis, population genetic and epidemiological methods, database curation/management, and reproducible research workflows</p></li>
<li><p><i class="fa fa-laptop-code"></i> Expert in R (tidyverse), RStudio, RMarkdown, Shiny, SLURM/HPC</p></li>
<li><p><i class="fa fa-laptop-code"></i> Experienced in Git/Github, LaTeX, CSS, HTML, Quarto, Python, REDCap</p></li>
</ul>
</div>
<div id="languages" class="section level2">
<h2>Languages</h2>
<ul>
<li>Native speaker English and Spanish</li>
<li>French Level A2</li>
<li>Dutch Level A1</li>
</ul>
</div>
<div id="training" class="section level2">
<h2>Additional training</h2>
<ul>
<li><strong>2021: Short course: Multiple imputation in <code>R</code></strong> (Victorian Centre for Biostatistics)</li>
<li><strong>2021: Diversity Scholar Workshop</strong> (RStudio)</li>
<li><strong>2018: Certification “Teaching Skills for Graduate Researchers</strong> (University of Melbourne)</li>
<li><strong>2017: Winter School in Mathematical Biology</strong> (The University of Queensland)</li>
<li><strong>2015: Short course: Spatial mapping and GIS <code>R</code> skills</strong> (University of Melbourne)</li>
</ul>
</div>
</div>
<div id="main" class="section level1">
<h1>Main</h1>
<div id="title" class="section level2">
<h2>Shazia Ruybal-Pesántez</h2>
<div id="genomic-epidemiologist-phd-mrc-centre-for-global-infectious-disease-analysis-imperial-college-london" class="section level3">
<h3>Genomic Epidemiologist (PhD) | MRC Centre for Global Infectious Disease Analysis, Imperial College London</h3>
<p>I am a genomic epidemiologist with expertise in population genetics, bioinformatics, and epidemiology of infectious diseases, particularly malaria. I am based at Imperial College London and am an Adjunct Associate Professor at Universidad San Francisco de Quito in Ecuador. For a list of my publications, see <a href="https://scholar.google.com.au/citations?user=E7dY4foAAAAJ&amp;hl=en">Google Scholar</a>.</p>
</div>
</div>
<div id="qualifications" class="section level2" data-icon="graduation-cap" data-concise="true">
<h2>Qualifications</h2>
<div id="the-university-of-melbourne" class="section level3">
<h3>The University of Melbourne</h3>
<p>PhD in Genetic Epidemiology</p>
<p>Melbourne, Australia</p>
<p>2018</p>
<!-- <i class="fa fa-award"></i> Melbourne International Engagement Award recipient -->
<!-- Thesis: [Genetic epidemiology of the *Plasmodium falciparum* reservoir of infection in Bongo District, Ghana](https://minerva-access.unimelb.edu.au/handle/11343/216521) -->
</div>
<div id="the-colorado-college" class="section level3">
<h3>The Colorado College</h3>
<p>BA in Biology</p>
<p>Colorado Springs, CO, USA</p>
<p>2012</p>
<!-- <i class="fa fa-award"></i> Presidential Scholarship & Jason Wilkes Memorial Prize in Biology recipient -->
</div>
</div>
<div id="selected-research-experience" class="section level2" data-icon="laptop">
<h2>Selected Research Experience</h2>
<div id="research-associate-dept-of-infectious-disease-epidemiology" class="section level3">
<h3>Research Associate, Dept of Infectious Disease Epidemiology</h3>
<p><a href="https://www.imperial.ac.uk/people/s.ruybal">MRC GIDA</a>, Imperial College London</p>
<p>London, UK</p>
<p>2022 - Present</p>
<ul>
<li>Developing novel methods and computational tools to analyse genomic data to support malaria molecular surveillance. <em>My projects include: </em></li>
<li><i class="fa fa-github" style="color:#AA336A"></i> <a href="mrc-ide.github.io/SIMPLEGEN"><strong><code>SIMPLEGEN</code></strong></a> R package to simulate malaria genetic epi to explore the “power” of genomic surveillance sampling designs</li>
<li><i class="fa fa-github" style="color:#AA336A"></i> <a href="https://shaziaruybal.shinyapps.io/DRpower_app"><strong><code>DRpower</code></strong></a> Shiny app (not yet public) to simplify the design of studies that monitor rapid diagnostic tests effectiveness for malaria diagnosis in the field</li>
<li><i class="fa fa-wrench" style="color:#AA336A"></i><font color= "#AA336A"> Skills used:</font> R, RMarkdown, Github, Shiny, Quarto, Python</li>
</ul>
</div>
<div id="visiting-scientist-institute-of-microbiology" class="section level3">
<h3>Visiting Scientist, Institute of Microbiology</h3>
<p>Universidad San Francisco de Quito</p>
<p>Quito, Ecuador</p>
<p>2021 - 2022</p>
<ul>
<li>Support the coordination and data analysis (ongoing) of a cohort study to understand vaccine and infection‐induced immunity to SARS‐CoV‐2</li>
<li>Support data analysis (ongoing) on the genomic epidemiology and phylogenetics of SARS‐CoV‐2 in Ecuador</li>
<li><i class="fa fa-wrench" style="color:#AA336A"></i><font color= "#AA336A"> Skills used:</font> REDCap, R, RMarkdown, Github, HPC/SLURM</li>
</ul>
</div>
<div id="consultant-epidemiologist" class="section level3">
<h3>Consultant Epidemiologist</h3>
<p>Freelance</p>
<p>Quito, Ecuador</p>
<p>2020</p>
<ul>
<li>Consulted for the Ministry of Foreign Affairs and Migration &amp; National COVID-19 Emergency Response Committee providing internal analyses on case/testing trends and epi modelling (importation dynamics, <em>Reff</em>)</li>
<li>Consulted for the Ecuadorian Red Cross National Blood Bank to establish SARS-CoV-2 sero-surveillance as part of the COVID-19 emergency response, delivered a surveillance <code>flexdashboard</code> (not public)</li>
<li><i class="fa fa-wrench" style="color:#AA336A"></i><font color= "#AA336A"> Skills used:</font> R, RMarkdown, Github, Dashboards</li>
</ul>
</div>
<div id="research-associate-population-health-and-immunity-division" class="section level3">
<h3>Research Associate, Population Health and Immunity Division</h3>
<p>Walter &amp; Eliza Hall Institute of Medical Research</p>
<p>Melbourne, Australia</p>
<p>2019 - 2022</p>
<ul>
<li>Supported field implementation and lead the overall analysis (ongoing) of a 12‐month longitudinal cohort study in Papua New Guinea</li>
<li>Work involved <code>flexdashboard</code> for routine updates and data viz of both operational logistics aspects and real-time epidemiological trends</li>
<li>Worked with large epidemiological datasets from a range of malaria, vector-borne diseases, and COVID-19 projects and led the data curation, management and analysis using reproducible workflows</li>
<li><i class="fa fa-wrench" style="color:#AA336A"></i><font color= "#AA336A"> Skills used:</font> Applied epi, R, RMarkdown, Github/Gitlab, Dashboards, Shiny</li>
</ul>
</div>
<div id="phd-researcherresearch-associate-bio21-institute" class="section level3">
<h3>PhD Researcher/Research Associate, Bio21 Institute</h3>
<p>University of Melbourne</p>
<p>Melbourne, Australia</p>
<p>2014 - 2019</p>
<ul>
<li>Applied genomic epidemiology approaches (bioinformatic and population genetic methods) using <code>R</code> to better understand the malaria transmission dynamics in Ecuador and Ghana</li>
<li><i class="fa fa-wrench" style="color:#AA336A"></i><font color= "#AA336A"> Skills used:</font> Applied epi, R, RMarkdown, Github, Data management/curation</li>
</ul>
</div>
</div>
<div id="positions" class="section level2" data-icon="suitcase" data-concise="true">
<h2>Positions</h2>
<div id="research-associate-imperial-college-london" class="section level3">
<h3>Research Associate, <em>Imperial College London</em></h3>
<p>N/A</p>
<p>N/A</p>
<p>2022-Present</p>
</div>
<div id="adjunct-associate-professor-universidad-san-francisco-de-quito" class="section level3">
<h3>Adjunct Associate Professor, <em>Universidad San Francisco de Quito</em></h3>
<p>N/A</p>
<p>N/A</p>
<p>2020-Present</p>
</div>
<div id="research-fellow-now-honorary-university-of-melbourne-walter-and-eliza-hall-institute-of-medical-research" class="section level3">
<h3>Research Fellow (now Honorary), <em>University of Melbourne, Walter and Eliza Hall Institute of Medical Research</em></h3>
<p>N/A</p>
<p>N/A</p>
<p>2019-Present</p>
</div>
<div id="honorary-research-fellow-burnet-institute-australian-centre-for-research-excellence-in-malaria-research-and-centre-for-the-elimination-of-neglected-tropical-diseases" class="section level3">
<h3>Honorary Research Fellow, <em>Burnet Institute, Australian Centre for Research Excellence in Malaria Research and Centre for the Elimination of Neglected Tropical Diseases</em></h3>
<p>N/A</p>
<p>N/A</p>
<p>2019-2022</p>
</div>
<div id="research-assistant-new-york-university-school-of-medicine" class="section level3">
<h3>Research Assistant, <em>New York University School of Medicine</em></h3>
<p>N/A</p>
<p>N/A</p>
<p>2012-2014</p>
</div>
</div>
<div id="selected-teaching-experience" class="section level2" data-icon="chalkboard-teacher">
<h2>Selected Teaching Experience</h2>
<div id="r-ladies-melbourne-inc" class="section level3">
<h3>R-Ladies Melbourne Inc</h3>
<p>Registered female-led charity, &gt;2300 members</p>
<p>Melbourne, Australia</p>
<p>2019-2022</p>
<ul>
<li>Leadership roles (Secretary, President) organising <a href="https://www.meetup.com/en-AU/rladies-melbourne/">monthly meetups</a> ranging from beginner to advanced <code>R</code> <a href="https://github.com/R-LadiesMelbourne">topics</a>. See sidebar for workshops I have (co-)led</li>
</ul>
</div>
<div id="guest-lecturer" class="section level3">
<h3>Guest lecturer</h3>
<p>N/A</p>
<p>Ecuador/Australia</p>
<p>2019-Present</p>
<ul>
<li>2019: Nossal Institute for Global Health MPH on Malaria and Child Health</li>
<li>2021: Universidad San Francisco de Quito MSc Innovation on COVID‐19 epidemiology, variant dynamics and implications for public health in Ecuador</li>
</ul>
</div>
<div id="student-supervision" class="section level3">
<h3>Student supervision</h3>
<p>University of Melbourne</p>
<p>N/A</p>
<p>2016-Present</p>
<ul>
<li>Supervised/mentored students to completion in honors/undergrad (3), MSc (1) and PhD (3, 2 ongoing) with projects in population genetics, seroepidemiology, and computational biology. I serve on 3 PhD committees at UoM</li>
<li>Actively involved in the supervision, mentorship and capacity‐building of researcher officers/technical staff in Papua New Guinea and Ecuador</li>
</ul>
</div>
</div>
<div id="notable-awards" class="section level2" data-icon="medal">
<h2>Notable Awards</h2>
<div id="covid-19-digital-grant-regional-collaborations-programme" class="section level3">
<h3>COVID-19 Digital Grant (Regional Collaborations Programme)</h3>
<p>Australian Academy of Sciences &amp; Dept of Industry, Science, Energy and Resources</p>
<p>Melbourne, Australia</p>
<p>Apr 2021</p>
<ul>
<li>Awarded A$10,000 as Chief Investigator for the collaborative project: <a href="https://www.science.org.au/news-and-events/news-and-media-releases/regional-research-set-get-digital-boost">Fit‐for‐purpose analytical tools to support COVID‐19 sero‐surveillance in Papua New Guinea</a>. This led to the development of <a href="shaziaruybal.shinyapps.io/covidclassifyr"><code>covidClassifyR</code></a></li>
</ul>
</div>
<div id="kellaway-excellence-award-in-education" class="section level3">
<h3>Kellaway Excellence Award in Education</h3>
<p>Walter and Eliza Hall Institute of Medical Research</p>
<p>Melbourne, Australia</p>
<p>Dec 2019</p>
<ul>
<li>Awarded to recognize the significant contribution to improve female representation in statistical bioinformatics through joint leadership of <a href="https://github.com/R-LadiesMelbourne">R‐Ladies Melbourne</a>, the R programming community for women in Australia</li>
</ul>
<!-- Selected Presentations and Media {data-icon=microphone} -->
<!-- -------------------------------------------------------------------------------- -->
<!-- ### [`covidClassifyR`: Streamlining data analysis pipelines to enhance the monitoring of COVID-19 in Papua New Guinea](https://shaziaruybal.github.io/covidClassifyR_useR2022/#1) -->
<!-- Poster at useR! 2022 conference -->
<!-- Virtual -->
<!-- 2022 -->
<!-- ### Panelist on "Lecciones aprendidas con variantes COVID‐19" -->
<!-- Universidad San Francisco de Quito -->
<!-- Quito, Ecuador -->
<!-- 2022 -->
<!-- - Panel hosted by USFQ & national newspaper El Universo "Lessons learned from COVID‐19 variants" recording (in Spanish) [here](https://www.youtube.com/watch?v=9TOPklkF5ZA) and [media release](https://www.eluniverso.com/noticias/ecuador/en-vivo-webinar-sobre-las-lecciones-aprendidas-con-las-variantes-del-covid-19-nota/) -->
<!-- ### TV interview "Ómicron ‐ la nueva variante que está afectando al mundo" -->
<!-- Visionarias EC National News Program -->
<!-- Quito, Ecuador -->
<!-- 2021 -->
<!-- - Segment "Omicron‐the new variant affecting the globe” recording (in Spanish) [here](https://youtu.be/utSVelCdcvU) -->
</div>
</div>
</div>
<div id="second-page-sidebar" class="section level1 aside extra-sidebar">
<h1>Second page sidebar</h1>
<div id="tools" class="section level2">
<h2>Digital tools</h2>
<ul>
<li><p><a href="mrc-ide.github.io/SIMPLEGEN"><code>SIMPLEGEN</code></a>: <code>R</code> package to simulate malaria genetic epidemiology</p></li>
<li><p><a href="shaziaruybal.shinyapps.io/covidClassifyR"><code>CovidClassifyR</code></a>: Shiny app to process and analyse COVID-19 serological data. See my <i class="fa fa-youtube" style="color:#AA336A"></i> <a href="https://shaziaruybal.github.io/covidClassifyR_useR2022/#1">useR! 2022 presentation</a></p></li>
<li><p><a href="https://covid19analytics.shinyapps.io/VaccinationScore/"><code>CovidVaccinationScore</code></a>: Shiny app to help individuals better understand their “priority” status during vaccine roll-out in Ecuador</p></li>
<li><p><a href="">Ecuador blood bank COVID-19 sero-surveillance <code>flexdashboard</code> (not open-source)</a>: presented anonymized/aggregated SARS-COV-2 seroprevalence data from monthly screening of blood donation samples</p></li>
<li><p><a href="">ICEMR PNG weekly <code>flexdashboard</code> (not open-source)</a>: presented operational data (e.g. follow-up rates) for study team to plan field activities, as well as prelim epi trends (e.g. malaria test positivity)</p></li>
</ul>
</div>
<div id="workshops" class="section level2">
<h2>Workshops</h2>
<p><strong>Crea tu dashboard en <code>R</code>: Visualizando datos del mundial FIFA (Oct 2022)</strong>: Presented at <a href="https://shaziaruybal.github.io/dashboard-mundialFIFA/#1">Brown Bag Seminar InnovaLab</a> at Institute of Tropical Medicine/University Cayetano Heredia Peru (in Spanish) for the MSc program</p>
<p><strong>Applied Malaria Molecular Surveillance workshop (Aug 2022)</strong>: Co-organized <a href="https://github.com/mrc-ide/AMMS2022">4‐day workshop</a> for African researchers on how to use <code>R</code>-based data analysis tools for malaria genomics</p>
<p><strong>Shiny app <code>covidClassifyR</code> workshop (March 2022)</strong>: Led <a href="https://shaziaruybal.github.io/covidClassifyR-workshop/">4‐day virtual workshop</a> for researchers in Papua New Guinea on how to use <code>covidClassifyR</code></p>
<p><strong>Automate your CV in <code>rmarkdown</code>: easy as 1, 2, <code>knit</code>! (Feb 2022)</strong>: Presented at <a href="https://shaziaruybal.github.io/automate-cv-rmd/#1">R-Ladies Melbourne meetup</a>, the <a href="https://www.youtube.com/watch?v=Fc1RwRskk08"><i class="fa fa-youtube" style="color:#AA336A"></i> recording</a> has been viewed &gt;5600 times</p>
<p><strong>STEM Workshop “Solve an outbreak and become a disease detective” (Sep 2019 &amp; 2022)</strong>: Co-led workshop at “It Takes a Spark” Conference teaching young girls (Y7-12) epidemiology and <code>R</code> skills <a href="https://github.com/R-LadiesMelbourne/2019-0912-It-takes-a-spark">(materials)</a></p>
<p><br>
<br>
<br></p>
</div>
<div id="disclaimer" class="section level2">
<h2>Disclaimer</h2>
<p>Made with <a href="https://github.com/rstudio/pagedown">pagedown</a>, <a href="https://github.com/shaziaruybal/application-appliedepi/"><i class="fa fa-github"></i> source code</a></p>
<p>See more detailed CV <a href="https://shaziaruybal.com/files/cv-vitae.pdf">here</a></p>
<p>Last updated on 2023-05-27</p>
</div>
</div>

<script>
(function() {
  var i, s, j, el, els;
  var create_el = function(type, className, content, isHTML) {
    var el = document.createElement(type);
    if (className) el.className = className;
    if (content) {
      if (isHTML) {el.innerHTML = content;} else {el.innerText = content;}
    }
    return el;
  }

  // replace h2 title with h1
  var title = document.getElementById('title').querySelector('h2');
  title.parentNode.replaceChild(create_el('h1', false, title.innerHTML, true), title);

  // add the class .aside to #aside
  el = document.getElementById('aside');
  if (el) el.className += ' aside';

  // tweak class names of sections, and add icons
  els = document.getElementById('main').querySelectorAll('.level2');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (i === 0 && el.id === 'title') continue;
    el.className += ' main-block';
    if (el.dataset['concise']) el.className += ' concise';
    // if there is no icon, add an icon:
    if (el.firstElementChild.firstChild && el.firstElementChild.firstChild.nodeName !== 'I') {
      s = el.dataset['icon'] || 'bookmark';
      el.firstElementChild.insertBefore(create_el('i', 'fa fa-' + s), el.firstElementChild.firstChild);
    }
  }

  // tweak class names of blocks in sections
  els = document.getElementById('main').querySelectorAll('.level3');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (el.parentNode.id === 'title') continue;
    el.className += ' blocks';
    el.innerHTML = '<div class="date"></div>' + '<div class="decorator"></div>' +
      '<div class="details"><header></header><div></div></div>' + el.innerHTML;
    var ps = el.querySelectorAll('p');
    // move the date paragraph to the date div
    if (ps.length >= 3) {
      s = ps[2].innerText;
      s = s === 'N/A' ? [] : s.split(' - ');
      el.removeChild(ps[2]);
      for (j = 0; j < s.length && j < 2; j++) {
        el.children[0].appendChild(create_el('span', false, s[j]))
      }
    }
    // move title, description, location to the details div
    (function(h) {
      h.appendChild(el.children[3]);
      var p = el.children[3];  // description
      if (p.innerText !== 'N/A') {
        h.appendChild(create_el('span', 'place', p.innerHTML, true));
      }
      el.removeChild(p);
      p = el.children[3];  // location
      if (p.innerText !== 'N/A') {
        s = create_el('span', 'location', p.innerHTML, true);
        s.innerHTML = '<i class="fa fa-map-marker-alt"></i> ' + s.innerHTML;
        h.appendChild(s);
      }
      el.removeChild(p);
    })(el.children[2].firstElementChild);
    // move the rest of content in a block into the last div of the details div
    s = el.children[2];
    while (j = s.nextSibling) {
      if (j.className === 'aside') { s = j; continue; }
      el.children[2].lastElementChild.append(j);
    }
  }
})();
</script>


  </body>
</html>
